testing/vcs/test_vcs_autot_axis_titles.py

```diff
@@ -18,8 +18,8 @@
 plot = sys.argv[3]
 x_over_y = sys.argv[4]
 if (x_over_y == '0.5'):
-    xSize = 400
-    ySize = 800
+    xSize = 250
+    ySize = 500
 else:
     xSize = 800
     ySize = 400
```

Comment:
@aashish24 I am making the window tall and narrow or long and thin so that ratio=autot is will be very different than the old behavior where the data is scaled with the window.